docs/process.md

```diff
@@ -117,7 +117,7 @@ Make the changes to the code, tests, and documentations as needed by the issue.
    * All new public APIs (methods, classes) are **documented with header comments**.
    * **Documentations are updated** when necessary, particularly when there are changes or additions to software design as well as user-facing features.
 
-1. Push your branch to your fork, or to the committer repo if you have push access.
+1. Push your branch to your fork, or to the committer repo if necessary.
```

Comment:
`only if necessary`

Assuming you mean that one should never push to the main repo unless it is justified (e.g. `dev-es6`).